doc/install.md

```diff
@@ -8,15 +8,21 @@ The easiest way to install Livepeer is by downloading the `livepeer` and `livepe
 ### Option 2: Build from source
 You can also build the executables from scratch.
 
-1. If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
+&ensp; 1\. If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
 
-2. You can fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal.
+&ensp; 2\. You can fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal.
 
-3. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
+&ensp; 3\. Make sure you have the necessary libraries installed:
 
-4. You can now run `PKG_CONFIG_PATH=~/compiled/lib/pkgconfig go build ./cmd/livepeer/livepeer.go` from the project root directory. To get latest version, `git pull` from the project root directory.
+* Linux: `apt-get update && apt-get -y install build-essential pkg-config autoconf nodejs gnutls-dev`
```

Comment:
Why does the goclient need nodejs?

Reply:
Ah good point, I think we copied this from an issue in another repo not related to livepeer.

Reply:
I think this was copied from the [Dockerfile](https://github.com/livepeer/go-livepeer/blob/master/docker/Dockerfile#L7) in the repo - the nodejs dependency might've been leftover from another Dockerfile that was being used as a base image for this [nodejs program that used to be used for verification](https://github.com/livepeer/verification-computation-archive/blob/master/Dockerfile). We can probably do some cleanup and remove it.

Reply:
I don't think that mentioned Dockerfile is being used anywhere, so probably it could be just removed.

Reply:
Shall I fix this up in my next PR? feels kind of silly to open a seperate PR for it.

Reply:
IMO there is no such thing as a PR that is too small to open 😃